src/environments.js

```diff
@@ -161,7 +162,8 @@ defineEnvironment("cases", {
             type: "align",
             align: "l",
             pregap: 0,
-            postgap: fontMetrics.metrics.quad,
+            // TODO(kevinb) get the current style
+            postgap: fontMetrics.metrics.getQuad(Style.TEXT),
```

Comment:
The parser doesn't seem to have any idea of what the current style is, but maybe it should.